Description: remove the file giving mocha options, as it's using deprecated stuff
Author: Julien Puydt
Forwarded: https://github.com/Rich-Harris/magic-string/issues/176

--- a/test/mocha.opts
+++ /dev/null
@@ -1 +0,0 @@
---compilers js:buble/register
\ No newline at end of file
